tests/Elasticsearch/Tests/ClientIntegrationTests.php

```diff
@@ -18,9 +19,32 @@
  */
 class ClientIntegrationTests extends \PHPUnit\Framework\TestCase
 {
+
+    /**
+     * @var $client Elasticsearch\Client
+     */
+    protected static $client;
```

Comment:
What about making it non-static a recreating it for each test in `setUp()`? (it probably does not matter here, but I think it is a good practice to keep tests independent)

Reply:
well it's just to not initialize the client in every test below. I'm not sure to understand why it makes the test dependent. If you don't want to use the global client, you can ever initialize a new one.

Reply:
oh sorry, I read your comment in a wrong way. Well yes we can initialiaze it in a `setUp()` method